src/07-even-more-loops.js

```diff
@@ -126,8 +185,13 @@ function findLargest(array) {}
  *  findEqual([ 0, 11, -2, 5 ], 9);
  *  //> false
  */
-function findEqual(array, selected) {}
-
+function findEqual(array, selected) {
+  for(let i=0;i< array.length;i++){
+    if(array.includes(selected))
```

Comment:
Again includes is doing the work here for us no our loop. Refactor this to check each individual element's equality against the selected value.